src/io/pti.jl

```diff
@@ -520,24 +603,24 @@ elements from a line of a PTI file and populate them into an `Array{String}`.
 Comments, typically indicated at the end of a line with a `'/'` character,
 are also extracted separately, and `Array{Array{String}, String}` is returned.
 """
-function _get_line_elements(line::AbstractString)::Array
-    if length(collect(eachmatch(r"'", line))) % 2 == 1
+function _get_line_elements(line::AbstractString)
+    if count(i->(i=="'"), line) % 2 == 1
         throw(Memento.error(_LOGGER, "There are an uneven number of single-quotes in \"{line}\", the line cannot be parsed."))
     end
 
-    comment_split = r"(?!\B[\'][^\']*)[\/](?![^\']*[\']\B)"
-    line_comment = split(line, comment_split, limit=2)
+    line_comment = split(line, _comment_split, limit=2)
     line = strip(line_comment[1])
     comment = length(line_comment) > 1 ? strip(line_comment[2]) : ""
 
-    split_string = r",(?=(?:[^']*'[^']*')*[^']*$)"
-    elements = [strip(element) for element in split(line, split_string)]
+    #elements = [strip(e) for e in split(line, _split_string)]
+    elements = split(line, _split_string)
```

Comment:
Because this call is happening on every line in the file even minor inefficiencies can add up.  Running re-allocating the array generated by `split` was non-negligible.  Applying a destructive strip "in-place" is also an option, however I found that it was not required, so skipping this hear saves a fair amount of space.